infra/terraform/templates/alb.tf

```diff
@@ -9,20 +9,21 @@ resource "aws_alb" "wellcomecollection_alb" {
   security_groups = [
     "${aws_security_group.https.id}",
     "${aws_security_group.http.id}",
-    "${aws_security_group.node_app_port.id}"
+    "${aws_security_group.node_app_port.id}",
+    "${aws_security_group.docker.id}"
   ]
 }
 
 resource "aws_alb_target_group" "wellcomecollection" {
   name     = "wellcomecollection"
-  port     = 3000
+  port     = 80
   protocol = "HTTP"
   vpc_id   = "${aws_vpc.wellcomecollection.id}"
 
   health_check {
-    healthy_threshold   = 2
-    unhealthy_threshold = 10
-    timeout             = 20
+    healthy_threshold   = 5
+    unhealthy_threshold = 2
+    timeout             = 5
```

Comment:
Does this make sense 5 seconds to trigger a timeout but then 30 to try again? or am I misreading that.